src/cascadia/WindowsTerminal/IslandWindow.cpp

```diff
@@ -261,6 +261,16 @@ void IslandWindow::Initialize()
 
     _rootGrid = winrt::Windows::UI::Xaml::Controls::Grid();
     _source.Content(_rootGrid);
+
+    // initialize the taskbar object
+    HRESULT hr = CoCreateInstance(CLSID_TaskbarList, NULL, CLSCTX_INPROC_SERVER, IID_PPV_ARGS(&_taskbar));

```

Comment:
Since you're using `wil`, you can use the wil CoCreateInstance helper.

```
if (auto taskbar = wil::CoCreateInstanceNoThrow(CLSID_TaskbarList))
{
   // ... do things to set up taskbar
   _taskbar = std::move(taskbar);
}
```